.changeset/search-tofu-panaeng.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-node': patch
+---
+
+Use of `TestPipeline.withSubject()` is now deprecated. Instead, use the `fromCollator`, `fromDecorator`, or `fromIndexer` static methods to instantiate a test pipeline. You may also use the class' `withCollator`, `withDecorator`, and `withIndexer` instance methods to build test pipelines that consist of multiple test subjects.
```

Comment:
maybe good to add something around expected time of deletion? E.g.

```
Use of `TestPipeline.withSubject()` is now deprecated and will be deleted in next main release
```

Reply:
I'm feeling a little non-committal on a timeline for actually removing it; I figure sometime in the (fairly distant) future, we can do one big sweep of `@deprecated` code, remove all instances, and do a single `major` bump to `2.0.0` that covers all of them.  As it is, I think this is the only one, so far.

Reply:
Make sense 👍